front/lib/api/llm/clients/anthropic/index.ts

```diff
@@ -75,7 +75,8 @@ export class AnthropicLLM extends LLM {
       thinking: this.thinkingConfig,
       system: prompt,
       messages,
-      temperature: this.temperature,
+      // Thinking isn’t compatible with temperature: `temperature` may only be set to 1 when thinking is enabled.
```

Comment:
What about `temperature === 0.5` and 
```
thinkingConfig = {
    type: "enabled",
    budget_tokens: 0
  }
```

Does it pass or fail ?

If it fails, let's specify it in the comment, if it passes, then we should update the value of `temperature` (not to 1)

Reply:
Never mind 😄